ts/utils/staking_utils.ts

```diff
@@ -95,4 +95,8 @@ export const stakingUtils = {
 
         return `${name.slice(0, 37).trim()}...`;
     },
+    // Internally we define an active pool with _some_ zrx staked for the next epoch.
```

Comment:
NIT: should update this comment with the new definition of active or remove it 😄

Reply:
oops (great point!!!!!!), will remove, comments are too much upkeep